src/vs/workbench/browser/web.main.ts

```diff
@@ -542,3 +543,33 @@ export class BrowserMain extends Disposable {
 		return UNKNOWN_EMPTY_WINDOW_WORKSPACE;
 	}
 }
+
+function getProductConfiguration(): IProductConfiguration {
```

Comment:
This is not a good location for the method. You depend on a `vscode-workbench-product-overrides` element which will only ever be present when our server and our web HTML runs, but for example not on github.dev. If at all this code should go into `src/vs/code/browser/workbench/workbench.ts` which is the specific entry point that gets triggered from our web UI.